src/samples/Uno.Themes.Samples/Uno.Themes.Samples.macOS/Info.plist

```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleName</key>
 	<string>Uno.Themes.Samples</string>
 	<key>CFBundleIdentifier</key>
-	<string>uno.platform.material</string>
+	<string>uno.platform.themes</string>
```

Comment:
I don't know why this changed but it needs to stay uno.platform.material for now :(